Lib/test/test_poll.py

```diff
@@ -130,7 +130,7 @@ def test_poll2(self):
         p = proc.stdout
         pollster = select.poll()
         pollster.register( p, select.POLLIN )
-        for tout in (0, 1000, 2000, 4000, 8000, 16000) + (-1,)*10:
+        for tout in (None, -1000, 0, 1000, 2000, 4000, 8000, 16000) + (-1,)*10:
```

Comment:
I'm not sure that it is correct to insert infinite timeouts before finite timeouts in this test. Add them after `16000`. And maybe correct the multiplier in `(-1,)*10`.